ert_shared/ensemble_evaluator/ensemble/prefect.py

```diff
@@ -184,31 +153,28 @@ def get_flow(self, ee_id, real_range):
             for iens in real_range:
                 transmitter_map[iens] = {
                     record: transmitter
-                    for record, transmitter in self.config[ids.INPUTS][iens].items()
+                    for record, transmitter in self._inputs[iens].items()
                 }
                 for step in self._reals[iens].get_steps_sorted_topologically():
                     inputs = {
                         inp.get_name(): transmitter_map[iens][inp.get_name()]
                         for inp in step.get_inputs()
                     }
-                    outputs = self.config[ids.OUTPUTS][iens]
-                    max_retries = self.config.get(ids.MAX_RETRIES, DEFAULT_MAX_RETRIES)
+                    outputs = self._outputs[iens]
                     # Prefect does not allow retry_delay if max_retries is 0
-                    if max_retries == 0:
-                        retry_delay = None
-                    else:
-                        delay = self.config.get("retry_delay", DEFAULT_RETRY_DELAY)
-                        retry_delay = timedelta(seconds=delay)
+                    retry_delay = None if self._max_retries == 0 else self._retry_delay
```

Comment:
I don't want this changed, but could this be done on `build()`?

Reply:
Yes we could! Question is whether we want this prefect specific settings quirk to reside in the builder or be hidden inside the prefect ensemble entirely.

Reply:
If it's possible to make a decision on build time, then I think the responsibility should lie with the builder.